docs/markdown/ios/card/card.header.md

```diff
@@ -1,41 +1,28 @@
 ---
 title: Card.Header
-description: Card.Header sits at the top, inside of the Card container. It is a required subcomponent that includes pre-text, a title, and navigational elements. The Card.Header should always, at the very least, include a title.
+description: Card.Header sits at the top, inside of the Card container. It is a required subcomponent that includes pre-text, a title, and navigational elements. Card.Header should always, at the very least, include a title.
 fullwidth: true
 ---
 
-<ImgContainer src="https://i.pinimg.com/originals/11/90/6b/11906b1575ba50e08d48b8a0bf20e805.png" alt="Example of a Card.Header with a highlighted area to show how it is placed and arranged in a Card." noPadding color="background-elevation-accent"/>
-
-
-## Usage guidelines
-
-<TwoCol>
-<Group>
-<Do title="When to use"/>
-Show a navigational arrow or more menu within the Card.Header to direct the user to the appropriate additional content.
```

Comment:
These two points aren't "when to use"/"when not to use", they're "Do"/"Don't"'s. Not sure if there are better usage guidelines to add here

Reply:
good catch! these are wrong. Should be: 
https://coda.io/d/_dqIC9SDPsU9/CardHeader-subcomponent_supz_#_luOB4

Then best practices are: https://coda.io/d/_dqIC9SDPsU9/CardHeader-subcomponent_supz_#_luPnt